### PR TITLE
Add another usage to example

### PR DESCRIPTION
**Pull request by nickmao1994**

### CODE REVIEW

File: docs/example.ipynb [hunk 1]
```diff
@@ -186,7 +186,7 @@
    "source": [
     "## Create correlation matrix\n",
     "\n",
-    "We can create a generic correlation matrix and a longer form one for all numerical variables in a data frame using the `corr_matrix` function."
+    "We can create a generic correlation matrix and a longer form one for all numerical variables in a data frame using the `corr_matrix` function. Use `[0]` to get a correlation matrix in its longer form and use `[0]` for a generic correlation matrix."
```
Comment (katerinkus) [inline docs/example.ipynb:189]: Hey Nick! Both options say that you should pick [0]. Perhaps, one of them should be [1]?

**Review by katerinkus — COMMENTED** [review]
Hey Nick, it seems you may have a typo:

File: docs/example.ipynb (hunk 1 above)
Comment (nickmao1994) [reply docs/example.ipynb]: Yeah, thanks for pointing this out!

Comment (katerinkus) [reply docs/example.ipynb]: Looks good!

**Pull request merged**